### ISSUE TITLE
Mode Collapse!

### ISSUE THREAD

**Issue by gamladz** [lucidrains/stylegan2-pytorch#223]
Hello,
![5](https://user-images.githubusercontent.com/4943670/113123321-619eec00-920c-11eb-8de1-f333e4066544.jpg)
![8](https://user-images.githubusercontent.com/4943670/113123333-6499dc80-920c-11eb-90da-c260c588ae79.jpg)

We're trying to use PyTorch on our furniture dataset but keep encountering mode collapse. Have attached the 5th and produced images. I've read through other folks strategies with the same issues and tried using the following settings.
 
 `stylegan2_pytorch --data PNG --multi-gpus --attn-layers [1,2] --batch-size 8 --gradient-accumulate-every 16`

Any ideas?

**Comment by lucidrains** [comment]
@gamladz more data and higher augmentation probability!

**Comment by thusinh1969** [comment]
StyleGAN current plus ADA whatever can only work with very portrait and focused-simple to detect dataset (faces). Do NOT expect any good results at all with other perplexed dataset. It is the capacity of network and its current design will NOT do the job.

have been trying a lot on models, twisted and trained for whatever resolution, GPU etc. it simply can not learn anything good but faces  !!!
Steve
